dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 requests==2.23.0
+pytest-asyncio == 0.15.1
+aiohttp==3.8.1
```

Comment:
I have already updated as first commit to the async-feature branch the dev-dependency.txt `pytest-aiohttp>=1.0.4`. 

Wouldn't explicit pytest plugin by better than generic aiohttp library? Just asking what dependency is better, doesn't seem right to have both.

Reply:
We don't have errors in our test, I forgot the requirements. We added our dev-requirements.txt. We don't have aiohttp because we want to stay generic without aiohttp dependency.

Reply:
I must say I am not understanding the last comment at all.

1. what does "We don't have aiohttp test because we want to stay generic without aiohttp dependency." mean? Especially when you commited  `aiohttp==3.8.1` to the dev-dependencies already? Do you want that or not?

2. And should the `pytest-aiohttp` from me, which already is in the `async-feature` branch dev-requirements.txt  stay or not? Should the async tests be based on pytest-aiohttp plugin or pytest-asyncio +aiohttp ? I expected we will go pytest plugins way.

Reply:
Question: is your code in client.py/service.py compatible with `httpx` python package? 
Or does it expect `aiohttp.ClientSession()` to be provided in the `Client.build_async_client` only?

If it is more open (best possible outcome, I don't actually know about any other than `requests` library being used with pyodata, see for example https://github.com/SAP/python-pyodata/pull/154#issuecomment-863462533 ), then why not add both integration tests with `aiohttp `and `httpx `packages?

Reply:
We should check, in theory it should work with httpx as well.